package.json

```diff
@@ -14,15 +14,27 @@
     },
     "license": "MIT",
     "author": "Laravel",
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
+    "exports": {
+        ".": {
+            "import": "./dist/index.mjs",
+            "require": "./dist/index.cjs",
+            "types": "./dist/index.d.ts"
+        },
+        "./inertia-helpers": {
+            "import": "./inertia-helpers/index.js",
+            "types": "./inertia-helpers/index.d.ts"
+        }
+    },
     "files": [
         "/dist",
         "/inertia-helpers"
     ],
     "scripts": {
         "build": "npm run build-plugin && npm run build-inertia-helpers",
-        "build-plugin": "rm -rf dist && tsc && cp src/dev-server-index.html dist/",
+        "build-plugin": "rm -rf dist && npm run build-plugin-types && npm run build-plugin-esm && npm run build-plugin-cjs && cp src/dev-server-index.html dist/",
+        "build-plugin-types": "tsc --emitDeclarationOnly",
+        "build-plugin-cjs": "esbuild src/index.ts --platform=node --format=cjs --outfile=dist/index.cjs",
```

Comment:
```suggestion
        "build-plugin-cjs": "esbuild src/index.ts --platform=node --external:vite --format=cjs --outfile=dist/index.cjs",
```

Reply:
I don't think this is necessary. I just gave it a try and got the following error:

![image](https://user-images.githubusercontent.com/4977161/235810521-a396d5c5-5962-4cef-9bb9-9b124b44ae71.png)

I've also confirmed that the output does not include Vite itself, as this is really just a transpilation of a single file.